app/models/agents/digest_agent.rb

```diff
@@ -10,6 +10,8 @@ class DigestAgent < Agent
       The resulting Event will have a payload message of `message`. You can use liquid templating in the `message`, have a look at the [Wiki](https://github.com/cantino/huginn/wiki/Formatting-Events-using-Liquid) for details.
 
       Set `expected_receive_period_in_days` to the maximum amount of time that you'd expect to pass between Events being received by this Agent.
+
+      If `clear_queue` is set to 0, Events are automatically purged from the memory when an Event is emitted by this Agent. To maintain a fixed number of Events, set `clear_queue` to that number.
```

Comment:
@cantino That sounds right. @dsander If you have anything else, let me know, otherwise I'll do what @cantino said, and add:

>For instance, say `retained_events` is set to 3 and the Agent has received Events `A`, `B`, and `C`. After a digest is sent, Events `A`, `B`, and `C` are retained for a future digest. After event `D` is received, the next digest will contain `B`, `C`, and `D`.